libs/application/templates/party-letter/src/lib/messages.ts

```diff
@@ -104,7 +104,7 @@ export const m: MessageDir = {
     partyLetterSubtitle: {
       id: 'ple.application:partyletter.subtitle',
       defaultMessage:
-        'Við  alþingiskosningarnar 28. október 2017 buðu eftirtalin stjórnmálasamtök fram lista og voru þeir merktir sem hér segir:',
+        'Við  alþingiskosningarnar 28. október 2017 buðu eftirtalin stjórnmálasamtök fram lista og voru þeir merktir sem hér segir:',
```

Comment:
extra whitespace here?

Reply:
ahh just text copied from figma :) i will fix it